src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+
+  if (days <= 2) {
+    return days * price;
+  }
+
+  if (days <= 6) {
+    return days * price - 20;
```

Comment:
You can remove the 'else' after an 'if' block that contains a 'return' statement. The 'else' is unnecessary because if the condition is true, the function will return and the subsequent code won't be executed.